### PR TITLE
Fixup VMware module docs issues

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Fix various argspec vs documentation issue

(cherry picked from commit 5d0ed38e3ad3c4b83bb0c083a8832f8a926da31c)

##### ISSUE TYPE
 - Bugfix Pull Request
 - Docs Pull Request

##### COMPONENT NAME

various files

##### ANSIBLE VERSION

```
Stable-2.5
```

### CODE REVIEW

File: test/sanity/validate-modules/ignore.txt [hunk 1]
```diff
@@ -162,19 +162,30 @@ lib/ansible/modules/cloud/spotinst/spotinst_aws_elastigroup.py E323
 lib/ansible/modules/cloud/univention/udm_dns_zone.py E322
 lib/ansible/modules/cloud/univention/udm_share.py E322
 lib/ansible/modules/cloud/univention/udm_share.py E323
+lib/ansible/modules/cloud/univention/udm_share.py E324
```
Comment (mattclay) [inline test/sanity/validate-modules/ignore.txt]: This doesn't look right. The original PR only removed ignore entries, yet this one is adding them.

Comment (Akasurde) [reply test/sanity/validate-modules/ignore.txt]: OK.

**Comment by gundalow** [comment]
Looks good

**Review by gundalow — APPROVED** [review]

**Comment by Akasurde** [comment]
@gundalow @nitzmahone @mattclay Thanks for review and merge.

**Pull request merged**